src/components/SingleContent/SingleContent.css

```diff
@@ -0,0 +1,38 @@
+.media {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 5px;
+  margin: 5 px 0;
+  background-color: #282c34;
+  border-radius: 10 px;
+  position: relative;
```

Comment:
Just courious, what are you trying to achieve with the `relative`position?

Reply:
I changed in static, but relative is running to.